### PR TITLE
Delta conf fix

### PR DESCRIPTION
**Pull request by skp33**

### CODE REVIEW

File: src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala [hunk 1]
```diff
@@ -30,7 +30,7 @@ object DeltaSQLConf {
 
 
   val RESOLVE_TIME_TRAVEL_ON_IDENTIFIER =
-    buildConf("spark.databricks.timeTravel.resolveOnIdentifier.enabled")
```
Comment (tdas) [inline src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala]: @brkyvz does this make sense?

**Comment by brkyvz** [comment]
Hi @skp33 Good catches. These changes LGTM, but there is a merge conflict. Can you please fix that up?

**Comment by databricks-cla-assistant** [comment]
[CLA assistant check](https://cla.dev.databricks.com/delta-io/delta?pullRequest=26) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.databricks.com/delta-io/delta?pullRequest=26) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: skp33<br/>:x: Kaushal Prajapati<br/><hr/>**Kaushal Prajapati** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dev.databricks.com/check/delta-io/delta?pullRequest=26) it.</sub>

**Comment by mukulmurthy** [comment]
Hi Kaushal,

It looks like there's a license issue because you committed with a different email address. Can you please fix that (either by adding the email to GitHub or by amending your commit)?

**Comment by skp33** [comment]
Thanks @mukulmurthy, Fixed the license use.

**Comment by skp33** [comment]
@brkyvz resolved the merge conflict by reverting my second commit, because the fix which i have given is already present in master branch.

**Pull request closed without merging**